bundles/binding/org.openhab.binding.satel/src/main/java/org/openhab/binding/satel/internal/protocol/SatelModule.java

```diff
@@ -271,15 +271,15 @@ private SatelMessage readMessage() {
 
                 // if meanwhile thread has been interrupted, exit the loop
                 if (Thread.interrupted()) {
```

Comment:
I decided it would be better to throw InterrupedException in this case. No specific reason, in my opinion it's slightly better, but I am open to any solutions. :-)